pkg/ebpf/co_re_unsupported.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build !linux_bpf
+
+package ebpf
+
+import "errors"
+
+// GetBTFLoaderInfo is not supported without linux_bpf
+func GetBTFLoaderInfo() (string, error) {
+	return "", errors.New("GetBTFLoaderInfo not supported without linux_bpf")
```

Comment:
I'd prefer to not call out build tags in an error message
```suggestion
	return "", errors.New("BTF is not supported")
```